ouroboros-consensus/src/Ouroboros/Storage/ChainDB/API.hs

```diff
@@ -58,6 +54,12 @@ import           Ouroboros.Storage.FS.API.Types (FsError)
 import qualified Ouroboros.Storage.ImmutableDB as ImmDB
 import qualified Ouroboros.Storage.VolatileDB as VolDB
 
+-- Support for tests
+import           Ouroboros.Network.MockChain.Chain (Chain(..))
+import qualified Ouroboros.Network.MockChain.Chain as Chain
+
+type ReaderId = Int
```

Comment:
Can you move this to the  "Readers" section in this file (above/below the definition or `Reader`)?